openreview/venue/process/submission_process.py

```diff
@@ -94,7 +94,7 @@ def process(client, edit, invitation):
             writers=[venue_id],
             signatures=[venue_id],
             signatories=[venue_id, authors_group_id],
-            members=note.content['authorids']['value'] ## always update authors
+            members=list(set(note.content['authorids']['value'])) ## always update authors
```

Comment:
could you add a test that submits a paper with duplicated authors?

Reply:
done!